docs/resources/ip_sla_track_list.md

```diff
@@ -0,0 +1,177 @@
+
+* Supported in ACI versions: 4.1(1i) and later.
+
+* Distinguished Name Format: `uni/tn-{name}/tracklist-{name}`
+
+## GUI Information ##
+
+* Location: `Tenants -> Policies -> Protocol -> IP SLA -> Track Lists`
+
+## Example Usage ##
+
+The configuration snippet below creates a IP SLA Track List with only required attributes.
+
+```hcl
+
+resource "aci_ip_sla_track_list" "example_tenant" {
+  parent_dn = aci_tenant.example.id
+  name      = "test_name"
+}
+
+```
+The configuration snippet below shows all possible attributes of the IP SLA Track List.
+
+!> This example might not be valid configuration and is only used to show all possible attributes.
+
+```hcl
+
+resource "aci_ip_sla_track_list" "full_example_tenant" {
+  parent_dn       = aci_tenant.example.id
+  annotation      = "annotation"
+  description     = "description_1"
+  name            = "test_name"
+  name_alias      = "name_alias_1"
+  owner_key       = "owner_key_1"
+  owner_tag       = "owner_tag_1"
+  percentage_down = "30"
+  percentage_up   = "40"
+  type            = "weight"
+  weight_down     = "10"
+  weight_up       = "20"
+  relation_to_ip_sla_track_members = [
+    {
+      annotation = "annotation_1"
+      target_dn  = aci_ip_sla_track_member.example.id
+      weight     = "20"
+      annotations = [
+        {
+          key   = "key_0"
+          value = "value_1"
+        }
+      ]
+      tags = [
+        {
+          key   = "key_0"
+          value = "value_1"
+        }
+      ]
+    }
+  ]
+  annotations = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+  tags = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+}
+
+```
+
+All examples for the IP SLA Track List resource can be found in the [examples](https://github.com/CiscoDevNet/terraform-provider-aci/tree/master/examples/resources/aci_ip_sla_track_list) folder.
+
+## Schema ##
+
+### Required ###
+
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
+  - [aci_tenant](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/tenant) ([fvTenant](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvTenant/overview))
+* `name` (name) - (string) The name of the IP SLA Track List object.
+
+### Read-Only ###
+
+* `id` - (string) The distinguished name (DN) of the IP SLA Track List object.
+
+### Optional ###
+
+* `annotation` (annotation) - (string) The annotation of the IP SLA Track List object.
+  - Default: `orchestrator:terraform`
+* `description` (descr) - (string) The description of the IP SLA Track List object.
+* `name_alias` (nameAlias) - (string) The name alias of the IP SLA Track List object.
+* `owner_key` (ownerKey) - (string) The key for enabling clients to own their data for entity correlation.
+* `owner_tag` (ownerTag) - (string) A tag for enabling clients to add their own data. For example, to indicate who created this object.
```

Comment:
Would make the changes here based on the comments made for the data source.

Reply:
ok